ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -211,10 +212,12 @@ EOF
     done
     if [ $COUNTER -ne 100 ]; then
```

Comment:
Another suggestion, the `COUNTER` is used to control while loop, we should not reuse it and set it to 100 on line 209 as a flag. I think you can use `$STATUS` for the judgement.